assets/javascripts/discourse/components/data-explorer-bar-chart.js

```diff
@@ -68,31 +69,24 @@ export default Component.extend({
         ],
       },
     };
-  },
+  }
 
-  didInsertElement() {
-    this._super(...arguments);
-    this._initChart();
-  },
+  @bind
+  initChart(canvas) {
+    loadScript("/javascripts/Chart.min.js").then(() => {
+      const context = canvas.getContext("2d");
+      // eslint-disable-next-line
+      this.chart = new Chart(context, this.config);
+    });
+  }
```

Comment:
We could async-ify this:
```suggestion
  @bind
  async initChart(canvas) {
    await loadScript("/javascripts/Chart.min.js");
    const context = canvas.getContext("2d");
    // eslint-disable-next-line
    this.chart = new Chart(context, this.config);
  }
```